redux-excersice/src/components/FormComponent.js

```diff
@@ -1,8 +1,66 @@
-import React from "react";
+import React, { useCallback, useState } from "react";
+import { object, string, ref } from 'yup';
 
 export default function FormComponent() {
 
-  const handleSubmit = event => {}
+  const [values, setValues] = useState({
```

Comment:
Form values should come from useReducer state, not a not another layer of useState.